### PR TITLE
Skip system config updates without a database

### LINKED ISSUE #40: Starting with a new project fails due to missing database
It looks like we should check for any relevant table, before executing `scriptUpdateConfig`

image

### PR DESCRIPTION
**Pull request by jochenmanz**
### 1. Why is this change necessary?
To resolve #40 

### 2. What does this change do, exactly?
Adds a check for the `system_config` table

### 3. Describe each step to reproduce the issue or behaviour.
Start a project without this change and without a database

### 4. Please link to the relevant issues (if any).
#40 

### 5. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [x] I have written or adjusted the documentation according to my changes

### CODE REVIEW

File: devenv.nix [hunk 1]
```diff
@@ -62,10 +62,20 @@ let
...
     done
 
+    while ! [[ $($DEVENV_PROFILE/bin/mysql shopware -s -N -e 'SHOW DATABASES LIKE "shopware";') ]] ; do
+      ${pkgs.coreutils}/bin/sleep 1
+    done
+
+    TABLE=$(mysql shopware -s -N -e 'SHOW TABLES LIKE "system_config";')
+
+    if [[ $TABLE == "" ]]; then
+      echo "Table system_config is missing, skipping config update"
```
Comment (jinnoflife) [inline devenv.nix:72]:
```suggestion
      echo "Table system_config is missing, skipping config update."
      echo "You should run >updateSwConfig< to ensure the dev status of your setup!"
```

**Review by jinnoflife — CHANGES_REQUESTED** [review]

File: devenv.nix [hunk 2]
```diff
@@ -416,6 +426,8 @@ in {
         SHOPWARE_ES_INDEXING_ENABLED = "1";
         SHOPWARE_ES_HOSTS = "127.0.0.1";
         SHOPWARE_ES_THROW_EXCEPTION = "1";
+
+        RABBITMQ_NODENAME = "rabbit@localhost";
```
Comment (Wysselbie) [inline devenv.nix:430]: Avoiding DNS look ups.
```suggestion
        RABBITMQ_NODENAME = "rabbit@127.0.0.1";
```

File: devenv.nix (hunk 1 above)
Comment (jochenmanz) [reply devenv.nix]: added :)

File: devenv.nix (hunk 2 above)
Comment (jochenmanz) [reply devenv.nix]: good catch, added :)

**Review by jinnoflife — APPROVED** [review]

**Review by Wysselbie — APPROVED** [review]

**Pull request merged**